.github/workflows/docker_publish.yml

```diff
@@ -40,3 +40,35 @@ jobs:
           repository: ubercadence/server
           build_args: TARGET=auto-setup
           tags: ${{ github.event.release.tag_name }}-auto-setup, latestRelease-auto-setup
+  push_server_to_ghcr:
+    name: Push Docker server image to GitHub Container Registry
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Push to GitHub Container Registry (server build)
+        if: "!github.event.release.prerelease"
+        uses: docker/build-push-action@v1
+        with:
+          username: ${{ github.actor }}
+          password: ${{ secrets.CADENCE_SERVER_GHCR_PAT }}
```

Comment:
Need to change back to `GITHUB_TOKEN` once github supports this.
See [here](https://github.community/t/github-packages-not-showing-up-in-organization-packages-ui/130481/4) for more information